src/w3c/templates/cgbg-headers.js

```diff
@@ -10,10 +10,13 @@ export default (conf, options) => {
   if (existingCopyright) {
     existingCopyright.remove();
   }
+
+  const specTitleElem = document.querySelector("h1#title");
+  const specTitleElemClone = specTitleElem.cloneNode(true);
```

Comment:
Maybe merge these into one variable?

Reply:
I think both get used independently, but @janiceshiu can confirm... but it might be possible to remove one of the variables.

Reply:
Ah true, they are used independently. Sorry!